### PR TITLE
PIP-1237-dockerhub

### PR DESCRIPTION
**Pull request by ottojolanki**

### CODE REVIEW

File: get-splice-junctions.wdl [hunk 1]
```diff
@@ -2,13 +2,12 @@ version 1.0
 
 # ENCODE long read rna pipeline: get splice junctions
 
-#CAPER docker quay.io/encode-dcc/long-read-rna-pipeline:v1.3
-#CAPER singularity docker://quay.io/encode-dcc/long-read-rna-pipeline:v1.3
```
Comment (paul-sud) [inline get-splice-junctions.wdl:6]: You might want to add some usage documentation that `caper>=0.8.2.1` is required to autodetect docker image

File: long-read-rna-pipeline.wdl [hunk 2]
```diff
@@ -2,13 +2,13 @@ version 1.0
 
 # ENCODE long read rna pipeline
 
-#CAPER docker quay.io/encode-dcc/long-read-rna-pipeline:v1.3
-#CAPER singularity docker://quay.io/encode-dcc/long-read-rna-pipeline:v1.3
-#CROO out_def https://storage.googleapis.com/encode-pipeline-output-definition/longreadrna.output_definition.json
```
Comment (paul-sud) [inline long-read-rna-pipeline.wdl:7]: Similar to `get-splice-junctions.wdl`, there should probably be documentation that you need the latest caper for image autodetection.

**Review by paul-sud — CHANGES_REQUESTED** [review]
It looks like the workflow options still refer to the old quay images, should they?: https://github.com/ENCODE-DCC/long-read-rna-pipeline/blob/5a6264ed4ad5ce0068ab42819151e8196172cdf0/workflow_opts/docker.json

Also a question about moving the caper comments.

File: long-read-rna-pipeline.wdl (hunk 2 above)
Comment (ottojolanki) [reply long-read-rna-pipeline.wdl:7]: Thanks. Good points. Also noticed that the documentation is obsolete in few other points (such as statement that caper requires python 3.4 or newer.. I think nowadays it is 3.6).

File: get-splice-junctions.wdl (hunk 1 above)
Comment (ottojolanki) [reply get-splice-junctions.wdl:6]: Good point, added to the installation instructions.

**Review by paul-sud — APPROVED** [review]

**Pull request merged**